doc/version_history.rst

```diff
@@ -6,6 +6,17 @@
 Version History
 ###############
 
+v1.5.0
+======
+
+Prepare for salobj 7.
+
+Requires:
+
+* ts_salobj 7.0
+* ts_idl 3.1
+* IDL file for WeatherStation from ts_xml 8.0
```

Comment:
ts_xml 11 (it has the new config topics salobj 7 uses)

Reply:
Thanks! I updated the version mentioned there.